third_party/crypto/acme/acme.go

```diff
@@ -690,11 +798,7 @@ func (c *Client) retryPostJWS(ctx context.Context, key crypto.Signer, accountURL
 // postJWS signs the body with the given key and POSTs it to the provided url.
 // The body argument must be JSON-serializable.
 // The accountURL should be empty for account creation and certificate revocation.
-func (c *Client) postJWS(ctx context.Context, key crypto.Signer, accountURL, url string, body interface{}) (*http.Response, error) {
-	nonce, err := c.popNonce(ctx)
-	if err != nil {
-		return nil, err
-	}
+func (c *Client) postJWS(ctx context.Context, key crypto.Signer, accountURL, url, nonce string, body interface{}) (*http.Response, error) {
```

Comment:
Can you update the comments on every function/method you've changed the signature on to explain what the new parameter is, what it can be set to, and its behaviour if not set?

Reply:
Also, how come you've changed this to accept a nonce instead of just retrieving one as before?

Reply:
All in all, we should attempt to keep the diff in third_party *as minimal as possible* IMO 😄

Reply:
Yes, this signature shouldn't need to change I think. Some things moved around in dev but was moved back. Forgot about reverting this change